authentik/outposts/api/outposts.py

```diff
@@ -200,3 +203,18 @@ def default_settings(self, request: Request) -> Response:
         """Global default outpost config"""
         host = self.request.build_absolute_uri("/")
         return Response({"config": default_outpost_config(host)})
+
+    @permission_required(None, ["authentik_outposts.refresh_outpost"])
+    @extend_schema(
+        request=OpenApiTypes.NONE,
+        responses={
+            204: OpenApiResponse(description="Successfully refreshed outpost"),
+            400: OpenApiResponse(description="Bad request"),
+        },
+    )
+    @action(detail=True, methods=["POST"])
+    def force_refresh(self, request: Request, pk: int) -> Response:
+        """Force an outpost to refresh its configuration. Will also clear its cache."""
+        outpost: Outpost = self.get_object()
```

Comment:
since the action is POST, it'll require the permission `authentik_outposts.change_outpost` by default

Reply:
Oh yeah I added the permission but forgot to specify it, and to run the migrations 🤡

Reply:
Added the migration. How do you propose we handle the permissions thing?

Reply:
There was some way to specify that the "implied" permissions from the HTTP verb are not required, but I honestly cannot remember how that was done